### PR TITLE
Resolve packaging issues

### PR DESCRIPTION
**Pull request by eggplants**
- move all package information to setup.cfg
- add typed marker to declare SPARQLWrapper is a typed package

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -35,7 +35,8 @@ project_urls =
 [options]
 packages =
     SPARQLWrapper
-include_package_data = True
+include_package_data =
+    SPARQLWrapper = py.typed
```
Comment (aucampia) [inline setup.cfg:39]: [`include_package_data` is a boolean](https://setuptools.pypa.io/en/latest/userguide/datafiles.html), if you want to include specific files you should use [`package_data`](https://setuptools.pypa.io/en/latest/userguide/datafiles.html)
```suggestion
```

File: setup.cfg [hunk 2]
```diff
@@ -35,7 +35,8 @@ project_urls =
...
 install_requires =
     rdflib>=6.1.1
 python_requires = >= 3.7
```
Comment (aucampia) [inline setup.cfg:42]:
```suggestion
python_requires = >= 3.7

[options.package_data]
SPARQLWrapper = py.typed
```

File: setup.cfg (hunk 1 above)
Comment (aucampia) [reply setup.cfg:39]: see suggestion regarding `package_data`

**Review by aucampia — COMMENTED** [review]
Looks good, just need to use `package_data` instead of `include_package_data`

**Comment by eggplants** [comment]
Changes are duplicated with #201. So please close it after this pr is merged.

> Thank you for the PR, looks good, just need to use package_data instead of include_package_data

Thank you. Accepted your suggestions.

**Review by aucampia — APPROVED** [review]
Thanks for the PR, looks good, will merge on friday.

**Pull request merged**